packages/odyssey-react/src/components/Modal/Modal.tsx

```diff
@@ -126,15 +144,17 @@ export const Modal = withTheme(
           hidden={!open}
         >
           <div className={styles.overlay} tabIndex={-1}>
-            <div
-              className={styles.dialog}
-              role="dialog"
-              aria-modal="true"
-              aria-labelledby={modalHeadingId}
-              ref={modalDialog}
-            >
-              {children}
-            </div>
+            <FocusScope contain>
```

Comment:
Thoughts on building our own component like this that provides a public interface for all the hook behavior exposed through `utils` in this PR?

Reply:
If not, maybe we can export `FocusScope` from our utils package so we end up with a single entry point for the 3rd party dependency instead of having it sprinkled throughout the code.